server/src/main/java/com/datastrato/gravitino/server/web/ConfigServlet.java

```diff
@@ -46,19 +49,17 @@ public ConfigServlet(ServerConfig serverConfig) {
 
   @Override
   protected void doGet(HttpServletRequest req, HttpServletResponse res)
-      throws UnsupportedEncodingException, IllegalStateException, IOException {
+      throws IllegalStateException, IOException {
```

Comment:
Why we remove the  exception of `UnsupportedEncodingException`

Reply:
Because it's an IOException